### PR TITLE
Fix profiles

### PR DESCRIPTION
**Pull request by samlhao**
- The files are named ambiguously, so I added two profiles to run the consensus calling more clearly, e.g. `-profile artic,awsbatch` or `-profiles msspe,awsbatch`.
- removed references to old nextstrain steps
- removed unused scripts

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -9,7 +9,6 @@ params {
 
   ref = 'https://github.com/czbiohub/test-datasets/raw/msspe/reference/MN908947.3.fa'
   ref_host = 'http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz'
-  ref_gb = "https://github.com/czbiohub/test-datasets/raw/olgabot/msspe--add-human-and-kraken/reference/MN908947.3.gb"
```
Comment (jackkamm) [inline nextflow.config:12]: References to `ref_gb` also need to be removed from `main.nf` (this is what is causing the CI to fail)

File: nextflow.config [hunk 2]
```diff
@@ -26,8 +25,6 @@ params {
 
   minDepth = 10
 
-  clades = "https://github.com/czbiohub/test-datasets/raw/olgabot/msspe--add-human-and-kraken/testdata/data/clades.tsv"
```
Comment (jackkamm) [inline nextflow.config:29]: Delete `assignclades.py` as well, and remove the `--clades` argument from `alignment_assembly_stats.py`

File: nextflow.config [hunk 3]
```diff
@@ -48,17 +45,6 @@ params {
 // Analysis params
   sample_vcfs = false
 
-// Nextstrain data
```
Comment (jackkamm) [inline nextflow.config:51]: Delete `test_nextstrain.config` as well

Comment (jackkamm) [reply nextflow.config:51]: Also should delete `nextstrain-augur` from `environment.yaml`, the `nextstrain_ch` from `main.nf`, and references to `test_nextstrain.config` in `nextflow.config`

**Comment by samlhao** [comment]
Good point, I'll have this PR cover a general "fix profiles" purpose

**Pull request merged**